crates/virtio-queue/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Unreleased
+# v0.3.0
 
 ## Added
 - QueueStateT::size(): [#153](https://github.com/rust-vmm/vm-virtio/pull/153)
```

Comment:
can you also update these sections with the other things that were added/changed etc since 0.2.0?

Reply:
Oh right, didn't thoroughly verify everything is accounted for in the present changelog; will do that as well.